pydantic/_internal/_discriminated_union.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, discriminator: str, definitions: dict[str, core_schema.CoreSc
         # in the output TaggedUnionSchema that will replace the union from the input schema
         self._tagged_union_choices: dict[Hashable, core_schema.CoreSchema] = {}
 
-        # `_used` is changed to True after applying the discriminator to prevent accidental re-use
+        # `_used` is changed to True after applying the discriminator to prevent accidental reuse
```

Comment:
Lol not sure why we didn't think `reuse` was a word

Reply:
It's apparently common to use `re-use` as well, but let's comply with codespell here :)